packages/cardhost/app/templates/components/card-name-dialog.hbs

```diff
@@ -0,0 +1,16 @@
+<Dialog open={{true}} @title={{this.title}} @mode={{@mode}} @icon={{@icon}}>
+  <p class="dialog--message">
+    Give your card a meaningful name
+  </p>
+  <TextField
+    @id="card__name"
+    @placeholder="i.e. Millenial Puppies"
+    @setValue={{action this.updateCardName}}
```

Comment:
Optional - If you can just pass the function here and it works, that's better. When we turn on Octane linting, we'll have to remove all uses of "action."

Reply:
👍